### ISSUE TITLE
FxA telemetry

### ISSUE THREAD

**Issue by jvonitter** [MozillaReality/FirefoxReality#1610]
- [x] number of users who click the "sign in" button
- [x] number of users who successfully log-in to an account
- [x] number of users who sign out
- [ ] sign outs not caused by user manually signing out?
- [x] number of users who turn syncing off (for each bookmarks and history)
- [x] number of users who send a tab from FxR (across all users, all sessions)
- [x] number of users who receive a tab from another device (across all users, all sessions)
- [x] of users who have sent a tab ever, average number of tabs sent per week
- [x] of users who have received a tab ever, average number of tabs received per week
- [x] if possible - tab source: mobile/desktop percentage

------------

- [x] Do data review
- [x] Implement telemetry

**Comment by larsbergstrom** [comment]
It would be really excellent to fit this in for the FxA / Send to Device launch, if possible. lmk if I can help expedite data or legal reviews (ET has recently grown the data stewardship team, too). cc @jvonitter

**Comment by jvonitter** [comment]
cc @daoshengmu

**Comment by daoshengmu** [comment]
All of them can implement by Glean. There is no blocker issue for them because their lifetime looks like can be daily and no time measurement.

**Comment by daoshengmu** [comment]
One thing might be concerned is our FxR 6 release is blocked that makes a little risky for Glean because we haven't had a chance to check the Glean baseline metric with real users. Ideally, we need to make sure baseline data are match with our current DAU data first, then adding more telemetry on it.

**Comment by daoshengmu** [comment]
In terms of `bookmark/history`, we can reuse the telemetry from 
https://github.com/mozilla-mobile/android-components/blob/master/components/support/sync-telemetry/metrics.yaml.

Please take a look at `org_mozilla_vrbrowser.bookmarks_sync`, `org_mozilla_vrbrowser.history_sync` schema from our data source before we start.

**Comment by davismtl** [comment]
@irrationalagent do they need to track send tab on their end or do we have this convered?

**Comment by larsbergstrom** [comment]
If it's already covered on your end (via the component), then presumably there would already be data if we look at the metrics and filter for the Firefox Reality product, due to all of our internal testing?

**Comment by irrationalagent** [comment]
I am not seeing anything that looks like telemetry in the send tab android component, so I'm guessing it will not be automatically sending any telemetry out of the box
